docs/source/index.rst

```diff
@@ -42,3 +42,4 @@ Versions and ethers compatibility
    configuration.rst
    mock-contract.rst
    ens.rst
+   migration-guides.rst
```

Comment:
Can we change order:
   getting-started.rst
   compilation.rst
   basic-testing.rst
   matchers.rst
   fixtures.rst
   mock-contract.rst
   ens.rst
   configuration.rst
   migration-guides.rst